features/docs/getting_started.feature

```diff
@@ -9,7 +9,7 @@ Feature: Getting started
     When I run `cucumber`
     Then it should fail with:
       """
-      No such file or directory - features. You can use `cucumber --init` to get started.
+      Feature folder not found at
       """
```

Comment:
I think this message should be more detailed. Some possible solutions:
1. Use `Aruba.config.root_directory` and somehow use it in the string
2. Change the exception to report a relative path and not a absolute path (so that here we can write `tmp/features`
3. Change the exception to not report the path

Sorry for this, I haven't notice this test before. However, I think it's important to keep it as informative as possible.

What do you think?